packages/devextreme/js/__internal/ui/html_editor/html_editor.ts

```diff
@@ -57,17 +83,17 @@ class HtmlEditor extends Editor<Properties> {
 
   _aiDialog?: AIDialog;
 
-  _quillInstance?: any;
+  _quillInstance?: QuillInstance;
 
   _cleanCallback!: Callback;
 
   _contentInitializedCallback!: Callback;
 
-  _htmlConverter!: Converter;
+  _htmlConverter?: Converter;
```

Comment:
[excess change?]
For what? I see this property is always set but a bit later in constructor. I believe it was written this way to not duplicate option chaining operators you introduced now

Reply:
> this property is always set but a bit later in constructor

in constructor: 

```
_prepareHtmlConverter(): void {
    const { converter } = this.option();
    if (converter) {
      this._htmlConverter = converter;
    }
}
```

so this prop is not always set, cause in the options it looks like this:
```
_getDefaultOptions() {
    converter: null,
}
```